gpt_synthesizer/generate_main.py

```diff
@@ -0,0 +1,106 @@
+from langchain.chains import LLMChain
+from langchain.prompts.prompt import PromptTemplate
+import os
+from prompt import get_generate_main_prompt
+from gpt_synthesizer.parser import get_code_from_chat
+import logging
+from ui import GEN_MAIN_MSG
+
+
+def to_files(chat, chat_key):
```

Comment:
The same function exists in generate_code.py. Maybe import it instead of double definition.